CONTRIBUTING.md

```diff
@@ -377,8 +377,22 @@ A problem must have a unique slug. This slug is used as
   this **must not** be merged until the exercism/x-common PR is merged.
 
 ## Improving Consistency By Extracting Shared Test Data
-
-TODO: elaborate.
+The goal is to have a file `exercises/<slug>/canonical-data.json` for each exercise in this repository. This JSON file should contain the generic (i.e. language independent) test data for that exercise. With this the quality of the tracks can be improved, e.g. fixes or new tests can be propagated to all the tracks. They can then be implemented manually or by running the test generator. (See [this issue](https://github.com/exercism/todo/issues/13) for more background info.)
+
+The following steps help you extracting the canonical test data.
+
+1. Select the exercise for which you want to create the shared test data from the [list of open todos](https://github.com/exercism/todo/issues?q=is%3Aissue+is%3Aopen+label%3Ashared-test-data)
```

Comment:
For us (not blocking this PR), we should make sure that either this list has all the problems (what if some new problems were added since those issues were created and they have no canonical data?) or make something that makes http://x.exercism.io/v3/problems/data-todos easy to visualise and recommend that here instead of the issues.